gradle/libs.versions.toml

```diff
@@ -9,16 +9,16 @@ androidTargetSdk = "32"
 androidx-lifecycle = "2.6.1"
 androidx-navigation-compose = "2.5.1"
 coil = "2.2.1"
-composePlugin = "1.4.0"
+composePlugin = "1.5.0"
```

Comment:
but we can't update it without updating google compose to 1.5? And compose update at the main project is currently blocked